pol-core/clib/esignal.cpp

```diff
@@ -15,7 +15,24 @@ namespace Pol
 {
 namespace Clib
 {
-std::atomic<bool> exit_signalled( false );
+std::atomic<bool> signal( false );
+std::atomic<int> code = 0;
```

Comment:
I'd recommend _not_ wrapping the accessors in methods.  `exit_signalled` is checked a lot in the client I/O loops, for example, and semantically it's just a boolean that something might set.

Maybe those who are more up-to-date on C++ and atomics and will chime in. 

```suggestion
std::atomic<bool> exit_signalled( false );
std::atomic<int> signalled_exit_code( 0 );
```

Reply:
Addressed by removing the getter wrappers.